package.json

```diff
@@ -69,13 +74,22 @@
 		],
 		"globals": [
 			"globalThis"
-		]
+		],
+		"rules": {
+			"import/no-useless-path-segments": "off",
+			"unicorn/import-index": "off",
+			"import/extensions": "off"
+		}
 	},
 	"ava": {
+		"serial": true,
```

Comment:
Is this necessary? It will slow down testing and I think the ky tests can generally run concurrently.

Reply:
Good point